src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitRepoUrlParser.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.DotNet.DarcLib.Helpers;
+
+internal class GitRepoUrlParser
```

Comment:
```suggestion
internal static class GitRepoUrlParser
```

Reply:
why?

Reply:
You are declaring the intent that there's no value in instantiating this class so that users of this API know they can just expect static methods inside they can call.